--- clients/react/src/abstractChatBox.tsx
+++ clients/react/src/abstractChatBox.tsx
@@ -16,6 +16,7 @@
   FaPaperPlane,
   FaPause,
   FaPen,
+  FaPhoneAlt,
   FaPlay,
   FaStop,
 } from "react-icons/fa";
@@ -2326,7 +2327,7 @@
                   padding: 4,
                 }}
               >
-                📞
+                <FaPhoneAlt />
               </button>
             )}
           </div>
